frontend/lib/screens/results_screen.dart

```diff
@@ -213,57 +271,42 @@ class ResultsScreenState extends State<ResultsScreen> {
           } else if (snapshot.hasData) {
             receipt = snapshot.data!;
             return Container(
-                padding: EdgeInsets.only(top: 10, left: 50, bottom: 20),
-                child: Row(
-                  mainAxisSize: MainAxisSize.min,
-                  crossAxisAlignment: CrossAxisAlignment.center,
+                padding: EdgeInsets.only(top: 20, left: 20, bottom: 20),
+                child: Column(
+                  mainAxisAlignment: MainAxisAlignment.start,
+                  crossAxisAlignment: CrossAxisAlignment.start,
                   children: [
-                    Column(
-                      crossAxisAlignment: CrossAxisAlignment.start,
-                      children: [
-                        dropDown(),
-                        Row(
-                          children: [
-                            Icon(Icons.store),
-                            Padding(
-                                padding: EdgeInsets.only(left: 5),
-                                child: SizedBox(
-                                    width: sizedBoxWidth,
-                                    height: sizedBoxHeight,
-                                    child: Text(
-                                      receipt.recipient,
-                                      style: TextStyle(
-                                          fontSize: fontSize,
-                                          fontWeight: FontWeight.w600),
-                                    ))),
-                          ],
-                        ),
-                        Row(
-                          children: [
-                            Icon(Icons.date_range),
-                            Padding(
-                                padding: EdgeInsets.only(left: 5),
-                                child: Text(
-                                  DateFormat("yyyy-MM-dd")
-                                      .format(receipt.date)
-                                      .toString(),
-                                  style: TextStyle(
-                                      fontSize: fontSize,
-                                      fontWeight: FontWeight.w600),
-                                )),
-                          ],
-                        )
-                      ],
+                    Container(
+                        child: Text(
+                      receipt.recipient,
+                      style: GoogleFonts.roboto(fontSize: fontSize),
+                    )),
+                    Container(
```

Comment:
Perhaps date could be on the same row as store name? That would reduce the unused space on the right

Reply:
Currently, that would cause a problem for long store names. It should be possible by using rows, columns, and sized boxes, but I couldn't make it work in a nice way. But I agree the header can be improved.